.travis.yml

```diff
@@ -1,4 +1,8 @@
 language: node_js
 node_js:
-  - "0.8"
-  - "0.10"
\ No newline at end of file
+  - "node"
+  - "iojs"
```

Comment:
Thanks for the feedback :) I've made the changes, but having thought about it a bit more, I thought I'd get your opinion on the actual issue.

I was a bit confused at how to handle the situation. It's not so much `node 0.12` but `npm` shiped with the older versions doesn't support the `caret (^)` version in some of the dependencies in chalk.

I know that newer versions of `npm` work with older versions of `node`, but should I get travis to muck around with the `npm` version or, like you said, specify a `node 0.12` as the minimum version?

What do you think?